app/views/notification_mailer/digest.text.erb

```diff
@@ -0,0 +1,12 @@
+<% if @type == :digest %>
+  Your Notifications From The Past 24h
+<% elsif @type == :instant %>
+  Your Recent Notifications
+<% end %>
+
+<% @notifications.each do |project, project_notifications| %>
+  <% project_notifications.each do |_, notifications| %>
+    <% presenter = DigestPresenter.new(notifications, self) %>
+    <%= render partial: 'notification_mailer/notification', locals: { presenter: presenter } %>
```

Comment:
This isn't a plain text email. The partial loaded here has html formatting. I think we'll need a secondary partial for the plain text version.

Reply:
My mistake. I completely glossed over the  `app/views/notification_mailer/_notification.text.erb` file and thought we missed it. Sorry about that.

Reply:
Added one for the notification partial: https://github.com/dradis/dradis-ce/pull/526/files#diff-18372eaa9ef09e00d3ac3ab4bef3558d

and another for the comment partial: https://github.com/dradis/dradis-ce/pull/526/files#diff-7318e2e328e3ece858e4b0585ec04989